app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  class BaseController < ApplicationController
+    respond_to :json
+
+    def confirm_two_factor_authenticated_for_api
```

Comment:
should this be a base `before_action` so that subclasses don't need to do that explicitly?

Reply:
It only occurred to me now that this base controller is for any API, not specifically the verification API. I wonder if (a) we'll need shared behaviors across verification APIs or (b) we would want non-verification APIs that don't require an authenticated user. For the moment, this seems fine as-is, but I expect we could need to move it or add a new `Api::Verify::BaseController` ?